extensions/blocks/contact-form/components/jetpack-contact-form.js

```diff
@@ -270,18 +263,21 @@ class JetpackContactForm extends Component {
 							icon={ renderMaterialIcon(
 								<Path d="M13 7.5h5v2h-5zm0 7h5v2h-5zM19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16H5V5h14v14zM11 6H6v5h5V6zm-1 4H7V7h3v3zm1 3H6v5h5v-5zm-1 4H7v-3h3v3z" />
 							) }
+							instructions={ __(
+								'You’ll receive an email notification each time someone fills out the form. Where should it go, and what should the subject line be?',
+								'jetpack'
+							) }
 						>
 							<form onSubmit={ this.onFormSettingsSet }>
-								<p className="jetpack-contact-form__intro-message">{ this.getIntroMessage() }</p>
 								{ this.renderToAndSubjectFields() }
-								<p className="jetpack-contact-form__intro-message">
+								<div class="components-placeholder__instructions">
 									{ __(
 										'(If you leave these blank, notifications will go to the author with the post or page title as the subject line.)',
 										'jetpack'
 									) }
-								</p>
+								</div>
 								<div className="jetpack-contact-form__create">
-									<Button isPrimary type="submit" disabled={ this.hasEmailError() }>
+									<Button isDefault type="submit" disabled={ this.hasEmailError() }>
```

Comment:
Should this be isSecondary too? I guess we should only have one primary button per page?

Reply:
@scruffian as far as I can tell isDefault == isSecondary. For some reason isSecondary does not work on this block. If you have time to dig into why that is, that would be awesome. If not, no worries!

Reply:
Sure, but let's do it on another PR